### ISSUE TITLE
RadioButtonGroup rendering is broken in a window when setValue parameter is changed

### ISSUE THREAD

**Issue by ChrisKClark** [vaadin/framework#11201]
- Vaadin Framework version (8.5.2)
- Browser: Chrome Version 69.0.3497.92 (Official Build) (64-bit)
- Web container TomEE 1.7.2
- Description of the bug
A RadioButtonGroup will render correctly the first time a window is opened with the radioButtonGroup.setValue(someValue) being called. Subsequent calls to open the window with a different setValue() parameter will cause the radio button group to be partially rendered. 

- Minimal reproducible example:

   ```
    public class MyUI extends UI {
        @Override
        protected void init(VaadinRequest vaadinRequest) {
             boolean[] toggle = {false};
             TestWindow window = new TestWindow();
             Button button = new Button("open");
             button.addClickListener(ev -> {
                    addWindow(window);
                    toggle[0] = !toggle[0];
                    window.setSelection(toggle[0] ? "One" : "Two");
                });
                setContent(button);
        }

        class TestWindow extends Window {
            private RadioButtonGroup<String> group;

            public TestWindow() {
                group = new RadioButtonGroup<>();
                group.setItems("One", "Two");
                setContent(group);
            }

            public void setSelection(String selection) {
                group.setValue(selection);
            }
        }
    }

- Steps to reproduce:
1. Click button: popup window opens with correctly rendered radio button group.
2. Close popup window.
3. Click button: popup window opens with broken radio button group.

- Expected behavior: Should render the following for the radio button group on the second window opening:

```
<div tabindex="-1" class="v-select-optiongroup v-widget" style="outline-style: none;" role="radiogroup">
  <span class="v-radiobutton v-select-option v-select-option-selected">
    <input type="radio" name="gwt-uid-3" value="on" id="gwt-uid-4" tabindex="0">
    <label for="gwt-uid-4">One</label>
  </span>
  <span class="v-radiobutton v-select-option">
    <input type="radio" name="gwt-uid-3" value="on" id="gwt-uid-5" tabindex="0" checked="">
    <label for="gwt-uid-5">Two</label>
  </span>
</div>
```

- Actual behavior: I am seeing the following rendered for the radio button group on the second window opening:

```
<div tabindex="-1" class="v-select-optiongroup v-widget" style="outline-style: none;" role="radiogroup">
  <span class="v-radiobutton v-select-option">
    <input type="radio" name="gwt-uid-7" value="on" id="gwt-uid-8" tabindex="0">
    <label for="gwt-uid-8"></label>
  </span>
</div>
```

**Comment by TatuLund** [comment]
Please this another issue report, could it be about the same issue https://github.com/vaadin/framework/issues/11143 There is fix already, that you can test using 8.6-SNAPSHOT version.

**Comment by anasmi** [comment]
Hi,

as @TatuLund noticed, it looks like this issue is a duplicate of the one fixed and already merged to the master.

I will close this issue due to this, but, please, if the issue is still remains under any circumstances comment here and we will re-open it. 

Also, I have tested your example and can't reproduce the problem under 8.7-SNAPSHOT anymore : )
